labs/computing-experiment-metrics/metadata.md

```diff
@@ -0,0 +1,25 @@
+---
+title: Computing metrics with event-level experiment data
+summary: A notebook for computing "metric observations" with event-level experiment data
+revisionDate: '2020-09-15'
+labels:
+  - jupyter
+  - data
+  - analysis
+
```

Comment:
If you want to see this in draft on `optimizely.com` add an `excludeFromListing: true` parameter while you're working on this lab.